2200107/Q1.py

```diff
@@ -0,0 +1,75 @@
+# 최빈수 구하기
+
+# 비교용 list
+compli=[]
+
+# dictionary 사용하려 했으나 포기
+# compdic={}
+
+# cnt 담는 list. 빈도수 저장용
+newli=[]
+
+testli=[]
+
+T=int(input())
+for tc in range(1,T+1):
+    times=int(input())
+    score=list(map(int,input().split()))
+    print('-------------------------------------------------------------')
+    print('-------------------------------------------------------------')    
+    # 자꾸 range 에러나서 뭐가 문젠지 찾기 위해 작성한 test list
+    # score의 value로 만들었어야 했음. 리스트에 for를 몇 번이나 걸다보니 헷갈려 꼬인 이슈..
+    # for indx in score:
+    #     if indx not in testli:
+    #         testli.append(indx)
+    # print(testli)
+
+    # 빈도수를 담기 위해 newli의 index를 점수에 맞춰 101개로 만들었음. 0~100.
+    for e in range(101):
+        newli.append(0)
```

Comment:
```suggestion
    newli = [0] * 101
```
이런 식으로 작성해봐도 좋을 것 같아요!